### PR TITLE
6697 run default command formatting

### LINKED ISSUE #6697: test_auto3dseg_ensemble
**Describe the bug**
```
[2023-07-05T17:27:51.672Z] Starting test: test_ensemble (tests.test_auto3dseg_ensemble.TestEnsembleBuilder)...
[2023-07-05T17:27:52.745Z] 
  0%|          | 0/12 [00:00<?, ?it/s]
  8%|▊         | 1/12 [00:00<00:05,  2.06it/s]
 50%|█████     | 6/12 [00:00<00:00, 12.82it/s]
100%|██████████| 12/12 [00:01<00:00, 10.39it/s]2023-07-05 17:27:52,732 - INFO - BundleGen from https://github.com/Project-MONAI/research-contributions/releases/download/algo_templates/e01d67a.tar.gz
[2023-07-05T17:27:52.746Z] 
[2023-07-05T17:27:53.676Z] 
algo_templates.tar.gz: 0.00B [00:00, ?B/s]
algo_templates.tar.gz:   8%|▊         | 8.00k/94.3k [00:00<00:06, 13.7kB/s]
algo_templates.tar.gz: 96.0kB [00:00, 158kB/s]                             2023-07-05 17:27:53,357 - INFO - Downloaded: /tmp/tmplnggjmrc/algo_templates.tar.gz
[2023-07-05T17:27:53.676Z] 2023-07-05 17:27:53,357 - INFO - Expected md5 is None, skip md5 check for file /tmp/tmplnggjmrc/algo_templates.tar.gz.
[2023-07-05T17:27:53.676Z] 2023-07-05 17:27:53,357 - INFO - Writing into directory: /tmp/tmpke5jvc0n/workdir.
[2023-07-05T17:27:53.676Z] 2023-07-05 17:27:53,596 - INFO - Generated:/tmp/tmpke5jvc0n/workdir/dints_0
[2023-07-05T17:27:53.676Z] 2023-07-05 17:27:53,640 - INFO - Generated:/tmp/tmpke5jvc0n/workdir/segresnet_0
[2023-07-05T17:27:53.936Z] 2023-07-05 17:27:53,679 - INFO - segresnet2d_0 is skipped! SegresNet2D is skipped due to median spacing of [1.0, 1.0, 1.0],  which means the dataset is not highly anisotropic, e.g. spacing[2] < 3*(spacing[0] + spacing[1])/2) .
[2023-07-05T17:27:53.937Z] 2023-07-05 17:27:53,766 - INFO - Generated:/tmp/tmpke5jvc0n/workdir/swinunetr_0
[2023-07-05T17:27:53.937Z] 2023-07-05 17:27:53,780 - INFO - The keys num_warmup_epochs cannot be found in the /tmp/tmpke5jvc0n/workdir/dints_0/configs/hyper_parameters.yaml for training. Skipped overriding key num_warmup_epochs.
[2023-07-05T17:27:53.937Z] 2023-07-05 17:27:53,780 - INFO - The keys use_pretrain cannot be found in the /tmp/tmpke5jvc0n/workdir/dints_0/configs/hyper_parameters.yaml for training. Skipped overriding key use_pretrain.
[2023-07-05T17:27:53.937Z] 2023-07-05 17:27:53,781 - INFO - The keys pretrained_path cannot be found in the /tmp/tmpke5jvc0n/workdir/dints_0/configs/hyper_parameters.yaml for training. Skipped overriding key pretrained_path.
[2023-07-05T17:27:53.937Z] 2023-07-05 17:27:53,781 - INFO - The keys determ cannot be found in the /tmp/tmpke5jvc0n/workdir/dints_0/configs/hyper_parameters.yaml for training. Skipped overriding key determ.
[2023-07-05T17:28:33.591Z] 
dints_0 - training ...:   0%|          | 0/2 [00:00<?, ?round/s]
dints_0 - training ...:  50%|█████     | 1/2 [00:13<00:13, 13.30s/round]
dints_0 - training ...: 100%|██████████| 2/2 [00:28<00:00, 14.24s/round]
dints_0 - training ...: 100%|██████████| 2/2 [00:28<00:00, 14.10s/round]dints_0 - validation at original spacing/resolution
[2023-07-05T17:28:33.592Z] 
[2023-07-05T17:28:34.153Z] 2023-07-05 17:28:34,037 - WARNING - dints_0 - training: finished
[2023-07-05T17:28:44.100Z] Traceback (most recent call last):
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 354, in _check_seekable
[2023-07-05T17:28:44.101Z]     f.seek(f.tell())
[2023-07-05T17:28:44.101Z] AttributeError: 'bool' object has no attribute 'seek'
[2023-07-05T17:28:44.101Z] 
[2023-07-05T17:28:44.101Z] During handling of the above exception, another exception occurred:
[2023-07-05T17:28:44.101Z] 
[2023-07-05T17:28:44.101Z] Traceback (most recent call last):
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/train.py", line 27, in <module>
[2023-07-05T17:28:44.101Z]     fire.Fire()
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/fire/core.py", line 141, in Fire
[2023-07-05T17:28:44.101Z]     component_trace = _Fire(component, args, parsed_flag_args, context, name)
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/fire/core.py", line 475, in _Fire
[2023-07-05T17:28:44.101Z]     component, remaining_args = _CallAndUpdateTrace(
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/fire/core.py", line 691, in _CallAndUpdateTrace
[2023-07-05T17:28:44.101Z]     component = fn(*varargs, **kwargs)
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/train.py", line 23, in run
[2023-07-05T17:28:44.101Z]     run_segmenter(config_file=config_file, **override)
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/segmenter.py", line 2021, in run_segmenter
[2023-07-05T17:28:44.101Z]     run_segmenter_worker(0, config_file, kwargs)
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/segmenter.py", line 1992, in run_segmenter_worker
[2023-07-05T17:28:44.101Z]     segmenter = Segmenter(config_file=config_file, config_dict=override, rank=rank, global_rank=global_rank)
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/segmenter.py", line 535, in __init__
[2023-07-05T17:28:44.101Z]     self.model = self.setup_model(pretrained_ckpt_name=config["pretrained_ckpt_name"])
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/segmenter.py", line 647, in setup_model
[2023-07-05T17:28:44.101Z]     self.checkpoint_load(ckpt=pretrained_ckpt_name, model=model)
[2023-07-05T17:28:44.101Z]   File "/tmp/tmpke5jvc0n/workdir/segresnet_0/scripts/segmenter.py", line 870, in checkpoint_load
[2023-07-05T17:28:44.101Z]     checkpoint = torch.load(ckpt, map_location="cpu")
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 791, in load
[2023-07-05T17:28:44.101Z]     with _open_file_like(f, 'rb') as opened_file:
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 276, in _open_file_like
[2023-07-05T17:28:44.101Z]     return _open_buffer_reader(name_or_buffer)
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 261, in __init__
[2023-07-05T17:28:44.101Z]     _check_seekable(buffer)
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 357, in _check_seekable
[2023-07-05T17:28:44.101Z]     raise_err_msg(["seek", "tell"], e)
[2023-07-05T17:28:44.101Z]   File "/opt/conda/lib/python3.8/site-packages/torch/serialization.py", line 350, in raise_err_msg
[2023-07-05T17:28:44.101Z]     raise type(e)(msg)
[2023-07-05T17:28:44.101Z] AttributeError: 'bool' object has no attribute 'seek'. You can only torch.load from a file that is seekable. Please pre-load the data into a buffer like io.BytesIO and try to load from it instead.
[2023-07-05T17:28:44.663Z] 
```

MONAI-postmerge/detail/MONAI-postmerge/1445/pipeline

could you please have a look? @mingxin-zheng

### PR DESCRIPTION
**Pull request by wyli**
Fixes #6697 

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/apps/auto3dseg/auto_runner.py [hunk 1]
```diff
@@ -525,7 +525,7 @@ def set_device_info(
         self.device_setting["MN_START_METHOD"] = mn_start_method
 
         if cmd_prefix is None:
-            cmd_prefix = os.environ.get("CMD_PREFIX")
+            cmd_prefix = os.environ.get("CMD_PREFIX", "")
```
Comment (mingxin-zheng) [inline monai/apps/auto3dseg/auto_runner.py:528]: Thanks @wyli . There seems another occurrence in:

https://github.com/Project-MONAI/MONAI/blob/9f5a55ba479772bb72616aebd23528b9cd63b407/monai/apps/auto3dseg/ensemble_builder.py#L456

Comment (wyli) [reply monai/apps/auto3dseg/auto_runner.py:528]: thanks Mingxin, this is revised now.

**Review by mingxin-zheng — APPROVED** [review]

**Comment by mingxin-zheng** [comment]
Thanks @wyli . It looks good to me.

**Pull request merged**